test/test_2d.jl

```diff
@@ -395,6 +395,43 @@ end
       end
     end
   end
+
+  @testset "Subcell limiting coefficients" begin
+    isdir(outdir) && rm(outdir, recursive=true)
+    run_trixi(joinpath(examples_dir(), "tree_2d_dgsem", "elixir_euler_sedov_blast_wave_sc_subcell.jl"), maxiters=10)
+
+    @timed_testset "do not reinterpolate" begin
+      # Create and test output without reinterpolation
+      @test_nowarn trixi2vtk(joinpath(outdir, "solution_000010.h5"), output_directory=outdir, reinterpolate=false)
+      outfilename = "solution_000010.vtu"
+      out_file = joinpath(outdir, outfilename)
+
+      # save output file to `artifacts` to facilitate debugging of failing tests
+      testname = "2d-treemesh-shockcapturing-subcell-no-reinterp"
+      cp(out_file, joinpath(artifacts_dir, testname * "-" * outfilename), force=true)
+
+      # remote file path is actually a URL so it always has the same path structure
+      remote_filename = "2d/treemesh/dgsem_sedov_subcell_no_interp_10.vtu"
+      ref_file = get_test_reference_file("dgsem_sedov_subcell_no_interp_10.vtu", remote_filename)
+      compare_point_data(out_file, ref_file)
+    end
+
+    @timed_testset "do reinterpolate" begin
+      # Create and test output without reinterpolation
+      @test_nowarn trixi2vtk(joinpath(outdir, "solution_000010.h5"), output_directory=outdir, reinterpolate=true)
+      outfilename = "solution_000010.vtu"
+      out_file = joinpath(outdir, outfilename)
+
+      # save output file to `artifacts` to facilitate debugging of failing tests
+      testname = "2d-treemesh-shockcapturing-subcell-reinterp"
+      cp(out_file, joinpath(artifacts_dir, testname * "-" * outfilename), force=true)
+
+      # remote file path is actually a URL so it always has the same path structure
+      remote_filename = "2d/treemesh/dgsem_sedov_subcell_interp_10.vtu"
+      ref_file = get_test_reference_file("dgsem_sedov_subcell_interp_10.vtu", remote_filename)
+      compare_cell_data(out_file, ref_file)
+    end
```

Comment:
Once I'm using `compare_point_data`, once `compare_cell_data`. the reason for that is:
```
julia> trixi_include(joinpath(examples_dir(), "tree_2d_dgsem", "elixir_euler_sedov_blast_wave_sc_subcell.jl"), maxiters=10)
julia> trixi2vtk("out/solution_*.h5", output_directory="out/", reinterpolate=false)
julia> compare_point_data("out/solution_000010.vtu", "out/solution_000010.vtu")
julia> compare_cell_data("out/solution_000010.vtu", "out/solution_000010.vtu")
BoundsError: attempt to access 0-element Vector{LightXML.XMLElement} at index [1]
Stacktrace:
 [1] getindex
   @ ./essentials.jl:13 [inlined]
 [2] get_data_section(vtk_file::VTKFile, section::String)
   @ ReadVTK ~/.julia/packages/ReadVTK/XpdR0/src/ReadVTK.jl:397
 [3] get_cell_data
   @ ~/.julia/packages/ReadVTK/XpdR0/src/ReadVTK.jl:417 [inlined]
 [4] compare_cell_data(out_filename::String, ref_filename::String; atol::Float64, rtol::Float64)
   @ Main ~/git/Trixi2Vtk.jl/test/test_trixi2vtk.jl:86
 [5] compare_cell_data(out_filename::String, ref_filename::String)
   @ Main ~/git/Trixi2Vtk.jl/test/test_trixi2vtk.jl:69
 [6] top-level scope
   @ REPL[23]:1
```
But with `reinterpolate=true` I get
```
julia> trixi_include(joinpath(examples_dir(), "tree_2d_dgsem", "elixir_euler_sedov_blast_wave_sc_subcell.jl"), maxiters=10)
julia> trixi2vtk("out/solution_*.h5", output_directory="out/", reinterpolate=true)
julia> compare_point_data("out/solution_000010.vtu", "out/solution_000010.vtu")
BoundsError: attempt to access 0-element Vector{LightXML.XMLElement} at index [1]
Stacktrace:
 [1] getindex
   @ ./essentials.jl:13 [inlined]
 [2] get_data_section(vtk_file::VTKFile, section::String)
   @ ReadVTK ~/.julia/packages/ReadVTK/XpdR0/src/ReadVTK.jl:397
 [3] get_point_data
   @ ~/.julia/packages/ReadVTK/XpdR0/src/ReadVTK.jl:444 [inlined]
 [4] compare_point_data(out_filename::String, ref_filename::String; atol::Float64, rtol::Float64)
   @ Main ~/git/Trixi2Vtk.jl/test/test_trixi2vtk.jl:142
 [5] compare_point_data(out_filename::String, ref_filename::String)
   @ Main ~/git/Trixi2Vtk.jl/test/test_trixi2vtk.jl:124
 [6] top-level scope
   @ REPL[29]:1
julia> compare_cell_data("out/solution_000010.vtu", "out/solution_000010.vtu")
```

Does that make sense? In other tests, it seems like `compare_point_data` works for reinterpolation and no reinterpolation.